src/lib/memory/MMU.cc

```diff
@@ -0,0 +1,36 @@
+#include "simeng/memory/MMU.hh"
+
+#include "simeng/OS/Constants.hh"
+namespace simeng {
+namespace memory {
+
+MMU::MMU(std::shared_ptr<Mem> memory, VAddrTranslator fn, uint64_t tid) {
+  memory_ = memory;
+  translate_ = fn;
+  tid_ = tid;
+};
```

Comment:
On this function and `bufferRequest` the `;` is not needed after `}`

Reply:
done